server/docs/design/block-persistence.md

```diff
@@ -3,64 +3,213 @@
 ## Table of Contents
 
 1. [Purpose](#purpose)
+1. [Terminology](#terminology)
+1. [Abstractions](#abstractions)
 1. [Goals](#goals)
```

Comment:
could we move Goals after purpoose? it makes sense to start with the purpose/objective and then move to specific goals.

Reply:
✅ Done.